subsys/bluetooth/audio/has.c

```diff
@@ -1209,5 +1353,9 @@ int bt_has_register(const struct bt_has_register_param *param)
 
 	registered = true;
 
+#if defined(CONFIG_BT_HAS_FEATURES_NOTIFIABLE)
+	k_work_submit(&features_work);
+#endif /* CONFIG_BT_HAS_FEATURES_NOTIFIABLE */
```

Comment:
There is not point in doing this, as a device is not allowed to modify it's GATT DB while connected, meaning that `bt_has_register` cannot succeed while connected, and thus we cannot send any notifications